demos/build.gradle

```diff
@@ -1,10 +1,10 @@
 plugins {
-  id 'org.akhikhl.gretty' version '2.0.0' apply false
+  id 'org.gretty' version '2.1.0' apply false
 }
 
 subprojects {
   apply plugin: 'war'
-  apply plugin: 'org.akhikhl.gretty'
+  apply plugin: 'org.gretty'
```

Comment:
So far, without adding jcenter, I can't find the gretty jetty9 jar. So it's not working for me.
```
subprojects {
  apply plugin: 'war'
  apply plugin: 'org.gretty'

  repositories {
    jcenter()
  }

  gretty {
    httpPort = 8080
    contextPath = '/'
    servletContainer = 'jetty9'
  }

  dependencies {
    compile 'javax.servlet:javax.servlet-api:3.1.0', project(':impl')
    runtime 'ch.qos.logback:logback-classic:1.2.3', 'com.h2database:h2:1.4.196'
  }
}
```